CHANGELOG.md

```diff
@@ -16,9 +16,29 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 
+## [v2.0.0] - 2024-08-30
+
+### Added
+
+- `storage:schemes`, `storage:refs` and Storage Scheme Object
+- Support the storage extension in Links
+- Support for the Alternate Assets Extension
+- Support for other storage providers, including custom S3 hosts
+
+### Changed
+
+- The extension is a framework for storage providers, it doesn't strictly define the individual providers.
+- The storage providers are grouped in `storage:schemes` and located in the Item Properties, Collections or Catalog metadata
+- Assets and Links reference the storage schemes by key in `storage:refs`
+
+### Removed
+
+- `storage:platform`, `storage:region`, `storage:requester_pays` and `storage:tier`
```

Comment:
At first, this looked worrying, but looking at the README, it seems it is more a *change* to move the properties under *Storage Scheme Object*. Maybe a note should be added to be more explicit for users doing the migration from V1 to V2.

Reply:
Updated changelog